addon/mixins/run.js

```diff
@@ -319,12 +320,14 @@ export default Mixin.create({
      init() {
        this._super(...arguments);
 
-       this.pollTask((next) => {
+       let token = this.pollTask((next) => {
          this.get('api').request('get', 'some/path')
            .then(() => {
              this.runTask(next, 1800);
            })
-       }, 'foo-bar#watch-some-path');
+       });
+
+       this.set('pollToken', token);
```

Comment:
```
this._pollToken = token;
```

* should likely be private
* should likely not be change tracking aware

Reply:
Agree. Just adhering to the convention of using `set` always. Can change, as it's likely not destined for a template.

Reply:
ya, its likely safe for us to teach "private internal state" shouldn't ever be externally observable so it shouldn't ever use `set`